solc/jsonCompiler.cpp

```diff
@@ -68,13 +68,13 @@ ReadFile::Callback wrapReadCallback(CStyleReadFileCallback _readCallback = nullp
 				result.success = false;
 				result.contentsOrErrorMessage = "File not found.";
 			}
-			if (contents_c)
+			else if (contents_c)
```

Comment:
I actually think we should enforce the same behaviour of the CStyleCallback as ReadFile::Callback, of course that is a breaking change and cannot do it now.